v63003/outref/gtm8842.txt

```diff
@@ -7,7 +7,25 @@ File triggers.txt, Line 1: Added SET trigger on ^X named triggered
 1 trigger file entries did update database trigger content
 0 trigger file entries did not update database trigger content
 =========================================
-# Attempting to ZBREAK the trigger
+# Running test without restrict.txt (to show restricting TRIGGER_MOD is necessary)
+ls: cannot access '##SOURCE_PATH##/restrict.txt': No such file or directory
+# Attempting to ZBREAK the trigger (Expect a RESTRICTEDOP Error)
+# Setting off a trigger function with ZBREAK break points and ZSTEPS
+%YDB-I-BREAKZBA, Break instruction encountered during ZBREAK action
```

Comment:
Please set ZBREAK and ZSTEP actions to display that the ZBREAK and ZSTEP actions were executed and otherwise resume execution without breaking (i.e. do not want the BREAKZBA message byut instead want it to proceed executing).